<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>registerwindow</class>
 <widget class="QDialog" name="registerwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>55</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Digital Wallet</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_4">
     <property name="minimumSize">
      <size>
       <width>689</width>
       <height>135</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>33</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Username must be between 5 and 16 letters in length.
Password must be between 8 and 16 letters length</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignHCenter">
    <widget class="QLabel" name="label_5">
     <property name="minimumSize">
      <size>
       <width>56</width>
       <height>136</height>
      </size>
     </property>
     <property name="font">
      <font>
       <pointsize>22</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: darkred</string>
     </property>
     <property name="text">
      <string>Message: No Errors</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>User Name</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Password</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Show Pass</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>Register</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
